.github/workflows/build-latest.yml

```diff
@@ -50,3 +50,36 @@ jobs:
             { "network": "pubnet", "options": "--enable-horizon-captive-core --enable-core-artificially-accelerate-time-for-testing" },
           ]
         }
+
+  arm64:
+    uses: ./.github/workflows/build.yml
+    secrets:
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+    with:
+      arch: arm64
+      tag: latest-arm64
+      core_ref: v19.7.0
```

Comment:
is worthwhile to define these once at top in workflow env variables rather than dups in each arch job?

Reply:
Unfortunately the `env` context is not available inside calls to other workflows, so there's no way in the current version of GitHub Action's syntax to move these values into a single location.

Reply:
Actually, we could move this into a shared workflow now that all images build for all amd64/arm64. I'll look at doing that in afollow up PR as it is a refactor.

Reply:
sounds good if you think it's worthwhile on that refactor, thanks for further insight on .env usage, gtk, so, can 't do:

```
env: 
  CORE_REF: "xyz"
jobs:
  amd64:
    with:
      core_ref: ${{ env.CORE_REF }}
```

the `with` params don't get interpolated prior, just on the runner.

Reply:
That's right. You can use the env context inside steps, but when calling out to another work flow that happens not inside a step.